### PR TITLE
feat: create lambda to fetch pages

### PR DESCRIPTION
**Pull request by fampinheiro**

### CODE REVIEW

**Review by johnytiago — CHANGES_REQUESTED** [review]

File: apps/asap-server/test/handlers/pages/fetch.test.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import { APIGatewayProxyResult } from 'aws-lambda';
+import { handler } from '../../../src/handlers/pages/fetch';
+import { apiGatewayEvent } from '../../helpers/events';
+
+describe('Get /pages/{path+}', () => {
```
[the remark this answers is not in the document]
Comment (fampinheiro) [reply apps/asap-server/test/handlers/pages/fetch.test.ts:5]: It's a catch all, everything from the path is captured in that parameter.

File: apps/asap-server/test/handlers/pages/fetch.test.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  afterEach(() => {
```
[the remark this answers is not in the document]
Comment (fampinheiro) [reply apps/asap-server/test/handlers/pages/fetch.test.ts:6]:
```suggestion
```

File: apps/asap-server/test/handlers/pages/fetch.test.ts [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    jest.clearAllMocks();
+  });
```
Comment (fampinheiro) [inline apps/asap-server/test/handlers/pages/fetch.test.ts:8]:
```suggestion
```

File: packages/services-common/src/squidex/client.ts [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      form: {
+        grant_type: 'client_credentials',
+        scope: 'squidex-api',
+        client_id: squidex.clientId,
+        client_secret: squidex.clientSecret,
+      },
+    }).json();
+
+    token = res.then((r) => (r as { access_token: string }).access_token);
+    /* eslint-enable @typescript-eslint/camelcase */
```
Comment (johnytiago) [inline packages/services-common/src/squidex/client.ts:24]:
```suggestion
    /* eslint-enable @typescript-eslint/camelcase */
    token = res.then((r) => (r as { access_token: string }).access_token);
```

**Review by johnytiago — CHANGES_REQUESTED** [review]

File: packages/services-common/src/squidex/client.ts (hunk 4 above)
Comment (fampinheiro) [reply packages/services-common/src/squidex/client.ts]: I need the disable on the promise on line 23.

Comment (johnytiago) [reply packages/services-common/src/squidex/client.ts]: You should disable the rule for the line after. Not the line before

**Pull request merged**